src/opencga-data-upload.sh

```diff
@@ -15,24 +15,22 @@
 # See https://documentation.dnanexus.com/developer for tutorials on how
 # to modify this file.
 
-# configurations
-opencga_file_id="project-G5XXyY84XjQPZKJfKX0ZQZpz:file-G6XPFF04XjQP8Z175fG1q8Y7"
-
 main() {
 
     echo 'Value of input_vcf: "${input_vcf[@]}"'
     echo "Value of input_config: '${input_config}'"
+    echo "Value of input_config: '${input_cli}'"
```

Comment:
I think the download of the CLI needs to be transparent to the users so that they don't have to specify what is the file containing the CLI in the applet. This is why I downloaded the file using the file ID (see line 19) in develop. What do you think?